### PR TITLE
chore: update api

### PR DESCRIPTION
**Pull request by AuHau**
It is bit bundled PR with some small improvements, but main goal is to update the Reference API. See commits.

### CODE REVIEW

File: escape-chars.bash [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        sed -e '/^title:/n;s/\([^\\]\)\>/\1\\\>/g' "$file.back" &> "$file"
+        if [ $? -eq 0 ]; then
+            rm "$file.back"
+            echo "fixed: $file"
+        else
+            mv "$file.back" "$file"
+            echo "error: $file"
+        fi
+    fi
+done
```
Comment (vojtechsimetka) [inline escape-chars.bash:16]: Can you add info this file exists to the README?

**Review by vojtechsimetka — APPROVED** [review]
LGTM

**Comment by vojtechsimetka** [comment]
FYI merging this as `chore:` means it does not trigger the `release-please` mechanism.

File: escape-chars.bash (hunk 1 above)
Comment (AuHau) [reply escape-chars.bash:16]: Thought I added, I guess I didn't.

Comment (AuHau) [reply escape-chars.bash:16]: I actually did ;-) 
https://github.com/ethersphere/bee-js-docs/pull/54/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R74

**Pull request merged**